tools/setup_main.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Setup script for PyPI - placed in the sdist
+
+from setuptools import setup
+
+
+setup(
+    name="pybind11",
+    packages=[
+        "pybind11",
+        "pybind11.include.pybind11",
+        "pybind11.include.pybind11.detail",
+        "pybind11.share.cmake.pybind11",
```

Comment:
But these aren't really Python packages. Can't we just tell Python to include these as directories?

Reply:
No, they are "packages" with "data". In order to be in `/pybind11`, all folders must be declared "packages", regardless of whether they have `__init__.py`'s in them or not. And since they don't have `__init__.py`, they don't show up automatically with `find_package_data`. This went through a lot of iterations until it worked... ;)

Reply:
Ugh. Ugly. But nothing we can do about that, then. Good catch/debugging, then ;-)